VTEX - GiftCard Hub API.json

```diff
@@ -2434,56 +2502,60 @@
+                                "id": "2001023",
+                                "productId": "2000492",
+                                "refId": "35994",
+                                "name": "insert name here",
+                                "value": 14.99,
+                                "price": 14.99,
                                 "quantity": 1,
                                 "shippingDiscount": 0,
-                                "discount": 0,
-                                "priceTags": []
+                                "discount": 2.5,
+                                "priceTags": [
+                                    {
+                                        "name": "insert name here",
+                                        "value": 0
+                                    }
+                                ]
                             }
                         ],
                         "grandTotal": 0,
-                        "discounts": 0,
-                        "shipping": 0,
+                        "discounts": 2.5,
+                        "shipping": 7.27,
                         "taxes": 0,
-                        "itemsTotal": 15
+                        "itemsTotal": 14.99
                     },
                     "clientProfile": {
-                        "email": "email@domain.com",
-                        "firstName": "John",
-                        "lastName": "Smith",
-                        "document": "10617764093",
-                        "documentType": "cpf",
-                        "phone": "+5521993759875",
+                        "email": "email@email.com.br",
+                        "firstName": "example",
+                        "lastName": "example",
+                        "document": "02906792063",
+                        "phone": "+551111111111",
                         "birthDate": "0001-01-01T00:00:00",
                         "isCorporate": false
                     },
                     "shipping": {
-                        "receiverName": "John Smith",
-                        "postalCode": "24417-246",
-                        "city": "Leblon",
+                        "receiverName": "example",
+                        "postalCode": "22250040",
+                        "city": "Rio de Janeiro",
                         "state": "RJ",
                         "country": "BRA",
-                        "street": "Rua Aidea Barreto do Couto",
-                        "number": "9678",
-                        "neighborhood": "",
-                        "complement": "",
-                        "reference": ""
+                        "street": "Praia de Botafogo",
+                        "number": "11",
+                        "neighborhood": "example",
+                        "complement": null,
+                        "reference": null
                     }
                 }
             },
             "Cart1": {
```

Comment:
This component is used in a request body, therefore it must abide by the OAS guidelines for request bodies, please include examples to each field.

See our [openapi template](https://github.com/vtex/openapi-schemas/blob/master/templates/VTEX%20-%20Template%20openAPI.jsonc)

Reply:
Corrigido.